crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -100,6 +100,19 @@ def knife_files
           filecontent_replace(file, "nova_dashboard", "horizon")
         end
       end
+
+      # find admin node and update target_platform
+      nodes_path = knife_path.join("nodes")
+      nodes_path.children.each do |file|
+        save = false
+        json = JSON.load(file.read)
+        if json["crowbar"] && json["crowbar"]["admin_node"]
```

Comment:
Style/Next: Use `next` to skip iteration. (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)